anomalib/core/callbacks/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 from .compress import CompressModelCallback
 from .model_loader import LoadModelCallback
+from .normalization import NormalizationCallback
```

Comment:
Yeah, I tried `AnomalyScoreNormalizationCallback` but found it too verbose. `ScoreNormalizationCallback` would be better, though maybe a bit vague. How about `OutputNormalizationCallback`, because it normalizes the outputs of the model?

Reply:
I changed it to `OutputNormalizationCallback`. Let me know what you think